todo.org

```diff
@@ -8,6 +8,9 @@ For some reason ~inf-clojure--set-repl-type~ is called in:
 
 Seems better to do this on the two different connection methods and then be done with it?
 
+** TODO do we need repl type in both source buffer and connection?
+   these can get out of sync and lead to confusing errors when closing a repl and opening a new one. It seems like we keep the repl-type in the source buffer to prevent a single ~(with-current-buffer (process-buffer proc) inf-clojure-repl-type)~
```

Comment:
I think it's not needed in the source buffer. In the interest of simplicity I think we should go back to the concept of a "current-repl" that users select explicitly and afterwards all commands just interact with it. Then there's no need to track any relationships between source buffers and REPLs, although you can allow for `buffer-local` mappings to REPLs for conveniences sake. Static mapping is the best mapping! :D

Reply:
Yeah I've removed it.